src/audio/Kconfig

```diff
@@ -114,6 +114,13 @@ config COMP_SRC
 	help
 	  Select for SRC component
 
+config COMP_STUBS
+	bool "Build all selected 3P components with stubs"
```

Comment:
Minor nit: it would not hurt to write "selected third-party (3P)" at least once somewhere. But  a minor nit, I think 3P is nowadays almost part of English language.

Reply:
Thanks @kv2019i for the "translation", I had no idea what "3P" meant :-) I just skipped that sentence.

On one hand I'm not a native speaker, on the other hand I'm far from the only one. I would probably have understood "3rdP" but to honest I don't think the Kconfig help is a place where this abbreviation has a benefit/cost ratio > 1

Reply:
Fair point, thanks @kv2019i will fix that